src/api/public/apidocs-new/paths/configuration.yaml

```diff
@@ -0,0 +1,59 @@
+get:
+  summary: Display the configuration of this Open Build Service instance
+  description: |
+    Display the configuration of this Open Build Service instance.
+  security:
+    - basic_authentication: []
+  responses:
+    '200':
+      description: >
+        OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/configuration.yaml'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+  tags:
+    - Configuration
+
+put:
+  summary: Update the configuration of this Open Build Service instance
+  description: |
+    Update the configuration of this Open Build Service instance.
+  security:
+    - basic_authentication: []
+  requestBody:
+    description: Configuration definition.
+    required: true
```

Comment:
```suggestion
```
The request body seems not to be required. Performing a PUT request without a request body returns a 200.